.github/lineage.yml

```diff
@@ -0,0 +1,6 @@
+---
+version: "1"
+
+lineage:
+  skeleton:
+    remote-url: https://github.com/cisagov/skeleton-generic.git
```

Comment:
This should be `skeleton-tf-module`, not `skeleton-generic`.

Reply:
See commit 827372d.